docs/Events/bpmn-events/import-done.md

```diff
@@ -4,9 +4,42 @@ title: import.done
 nav_order: 4
 description: import.done event
 permalink: /events/bpmn-events/import-done
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-21T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # import.done
+
+This event is fired by [importXML API](/general/api-reference/bpmn-js/importXML) with data containing `errors` and `warnings` after the import ends. This allows users to listen to whether the import of a diagram was successful or not.
+
+## Example Usage
+
+```javascript
+// modeler is an instance of BpmnJS Modeler.
+
+modeler.on('import.done', (data) => {
+
+  const { errors, warnings } = data; 
+
+  // check for errors while importing the diagram
+  if(errors && errors.length) {
+    console.log('Errors while importing the diagram raised:', errors);
+  }
+
+  // check for warnings while importing the diagram
+  if(warnings && warnings.length) {
+    console.log('Warnings while importing the diagram raised:', warnings);
+  }
+});
+```
+
+## Data
+
+The data fired with this event contains these fields:
+
+`errors`: _Array\<[String](#)\>_ <br>
```

Comment:
I don't think we emit `errors` field which is an array of string, but we actually emit an `error` field which is an instance of `Error` object?

https://github.com/bpmn-io/bpmn-js/blob/36e4f6113c5fc1e826894244c6da39636b75861c/lib/BaseViewer.js#L169